src/engine.ts

```diff
@@ -310,8 +320,22 @@ export default class Engine {
       this.roundBoosters[booster] = true;
     }
 
-    this.players = [];
+    // Shuffle tech tiles 
+    const techtiles = shuffleSeed.shuffle(Object.values(TechTile), this.map.rng());
+    for (const techTilePos of Object.values(TechTilePos)) {
+      this.techTiles[techTilePos] = { tile : techtiles[0], available : true } ;
+      techtiles.splice(0,1);
```

Comment:
Instead of that, you could do something like:

```
Object.values(TechTilePost).map( (pos, i) => {
  this.techTiles[pos] = {tile: techTiles[i], available: true};
}):
```

Reply:
much more elegant!